components/eamxx/src/share/grid/remap/coarsening_remapper.cpp

```diff
@@ -68,6 +68,25 @@ CoarseningRemapper (const grid_ptr_type& src_grid,
   scorpio::grid_read_data_array(map_file,"S",  -1,S_h.data(),       nlweights);
   scorpio::eam_pio_closefile(map_file);
 
+  // Offset the cols ids to match the source grid.
+  // Determine the min id among the cols array, we
+  // also add the min_dof for the grid.
+  // Note: The cols field is not guaranteed to have the
+  // min offset value, but the rows will be numbered 
+  // by 1 from least to greatest.  So we use row_gids to calculate
+  // the remap min.
+  int remap_min_dof = io_grid->get_num_global_dofs()*100;  // Really big INT
```

Comment:
This is not really safe. If the number of global dofs is by chance larger than 20M (which happens with ne1024), then the result of the RHS will overflow, causing min dof to be inited to some neg number (which will mess up the min logiv below).

I think the safest thing to do is to init the int with `std::numeric_limits<int>::max()`.

Reply:
agreed